Library/Formula/libjit.rb

```diff
@@ -0,0 +1,19 @@
+class Libjit < Formula
+  homepage "https://www.gnu.org/software/libjit/"
+  url "git://git.savannah.gnu.org/libjit.git"
+  version "0.1.3"
+
+  depends_on "gcc" => :build
```

Comment:
Use `fails_with` instead of depending. Also, we're unlikely to accept this if it doesn't build with clang.